kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/ColumnMappingSuite.scala

```diff
@@ -607,6 +609,12 @@ class ColumnMappingSuite extends AnyFunSuite with ColumnMappingSuiteBase {
     if (enableIcebergCompatV2) {
       metadata = metadata.withIcebergCompatV2Enabled
     }
+    if (!metadata.getConfiguration.containsKey(COLUMN_MAPPING_MAX_COLUMN_ID_KEY)) {
+      // A hack, if the metadata doesn't have max column ID in it,
```

Comment:
I added this hack because most test cases wanted to test no-updates, just want to confirm we are OK with this.